assets/js/media-upload.js

```diff
@@ -5,11 +5,24 @@ jQuery(document).ready(function($) {
 		
 	// Uploading files
 	var file_frame;
+	let $settings_wrapper;
```

Comment:
@alexmigf This needs to stay because we need the `$settings_wrapper` outside of `on` click scope so that it is properly updated on each '.wpo_upload_image_button' click. If I remove this and update the  `$settings_wrapper` to be let as you stated in the comment below, application will not work properly - it will replace the first language value with the second one when I update the second language value.